hooks/zfs/zfs

```diff
@@ -0,0 +1,14 @@
+# vim: set ft:sh:
+# shellcheck shell-sh
+
+copy_file "${zfs_cache#*=}" /root/etc/zfs/zpool.cache 600 0
+
+for module in \
+    zlib_deflate spl zavl zcommon \
+    znvpair zunicode zlua zfs icp
+do
+    copy_module "$module"
+done
```

Comment:
`copy_module` is smart enough to handle dependencies. No need to specify them explicitly. `copy_module zfs` should be enough.

You can check output of `modprobe -D zfs` to ensure that all dependencies are listed. You will get something like this:

```
insmod /lib/modules/5.12.1/kernel/fs/zfs/spl/spl.ko.xz
insmod /lib/modules/5.12.1/kernel/fs/zfs/icp/icp.ko.xz
insmod /lib/modules/5.12.1/kernel/fs/zfs/avl/zavl.ko.xz
insmod /lib/modules/5.12.1/kernel/fs/zfs/nvpair/znvpair.ko.xz
insmod /lib/modules/5.12.1/kernel/fs/zfs/zcommon/zcommon.ko.xz
insmod /lib/modules/5.12.1/kernel/fs/zfs/lua/zlua.ko.xz
insmod /lib/modules/5.12.1/kernel/fs/zfs/zstd/zzstd.ko.xz
insmod /lib/modules/5.12.1/kernel/fs/zfs/unicode/zunicode.ko.xz
insmod /lib/modules/5.12.1/kernel/fs/zfs/zfs/zfs.ko.xz
```

Reply:
I actually have `zfs` built directly in and hadn't tested the module portion yet! Thanks for checking.